.github/stale.yml

```diff
@@ -0,0 +1,63 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - "contrib: maybe good first bug"
+  - "contrib: good first bug"
+  - "contrib: welcome"
+  - "component: security"
+  - "neverstale"
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
```

Comment:
I would ignore those. I try to keep my backlog of things I want to work on to less than a page of results (25), so I stay assigned on bugs I really want to fix someday. Doesn't mean I will of course, but I'd prefer to keep them opened, and unassign myself the day I get tired of seeing them and admit I'll never fix them.

Reply:
Does everyone do that though? 😀

Reply:
Maybe not, but I think it should be the assignee's responsibility to close or unassign issues already assigned to them, not a bot.